### PR TITLE
add order with self-pickup delivery type

### PR DESCRIPTION
**Pull request by AndriyLymych**
Form:
image

Loading:
image

Result in db:
image

### CODE REVIEW

**Comment by kapoliub** [comment]
Screen Shot 2021-02-17 at 11 03 50
fix this codesmell pls

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Then open one more terminal and run `npm run cypress`.
   - Each component should have at least three files:
     - `index.js` where we export anything from the whole folder
     - `{component-name}.js` - file where component is located
-    - `{component-name}.styles.js` where all styles are located
+    - `{component-name}.self-pickup.styles.js` where all styles are located
```
Comment (kapoliub) [inline README.md:109]: why you change it?

File: src/containers/checkout/checkout-form/delivery/self-pickup/self-pickup.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <div className={styles.schedule}>
+          <p className={styles.scheduleData}>
+            {Object.values(SCHEDULE[language]).map(value =>
+              value === SCHEDULE[language].saturday ||
+              value === SCHEDULE[language].sunday ?
+                <div className={styles.scheduleItem}> {value} {CHECKOUT_TITLES[language].restDay} </div> :
+                <div className={styles.scheduleItem}> {value} {CHECKOUT_TITLES[language].workDay} </div>
+            )}
+          </p>
+          <p className={styles.scheduleItem + ' '+styles.addressTitle}>
```
Comment (kapoliub) [inline src/containers/checkout/checkout-form/delivery/self-pickup/self-pickup.js:35]: make styles like this `${somestyle}`

**Comment by kapoliub** [comment]
check your eslint(prettier) configs

File: src/containers/checkout/checkout-form/delivery/self-pickup/self-pickup.js (hunk 2 above)
Comment (AndriyLymych) [reply src/containers/checkout/checkout-form/delivery/self-pickup/self-pickup.js]: fixed

File: README.md (hunk 1 above)
Comment (AndriyLymych) [reply README.md]: fixed

**Review by Haytham1337 — APPROVED** [review]

**Review by kapoliub — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Pull request merged**